src/main/java/util/FileHelper.java

```diff
@@ -0,0 +1,109 @@
+package util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * A helper that manages file I/O.
+ */
+public class FileHelper {
+    private static final Logger logger = LogManager.getLogger(FileHelper.class.getName());
+
+    private static final String CHARSET = "UTF-8";
+    private static final String EMPTY_STRING = "";
+
+    private final String configFileDirectory;
+    private final String configFileName;
```

Comment:
Yes, if config files are not involved at all. I mean, this class may be used to write to any kind of file.